sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java

```diff
@@ -279,6 +279,19 @@ public ConfigurationClientBuilder configuration(Configuration configuration) {
         return this;
     }
 
+    /**
+     * Sets the retry policy that is used during construction of the service client.
+     *
+     * The default retry policy will be used if not provided.
```

Comment:
Wrap this in <p></p>, maybe link the default constructor.

Reply:
I delete this constructor instead of having a setter to set

Reply:
Because it is a builder pattern. Also we need to renaming all to getter and setter